Shared/packages.config

```diff
@@ -41,6 +41,6 @@
   <package id="System.ValueTuple" version="4.5.0" targetFramework="portable45-net45+win8+wpa81" />
   <package id="System.Xml.ReaderWriter" version="4.3.1" targetFramework="portable45-net45+win8+wpa81" />
   <package id="System.Xml.XDocument" version="4.3.0" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Xam.Plugins.Settings" version="4.1.0-beta" targetFramework="portable45-net45+win8+wpa81" />
+  <package id="Xam.Plugins.Settings" version="3.1.1" targetFramework="portable45-net45+win8+wpa81" />
```

Comment:
must be 4.1.0-beta

Reply:
> must be 4.1.0-beta

we have runtime error with this version & other projects use 3.1.1